Headers/Foundation/NSFileManager.h

```diff
@@ -325,6 +325,7 @@ GS_EXPORT_CLASS
 - (BOOL) changeCurrentDirectoryPath: (NSString*)path;
 - (BOOL) changeFileAttributes: (NSDictionary*)attributes
 		       atPath: (NSString*)path;
+- (BOOL) setAttributes:(NSDictionary *)attributes ofItemAtPath:(NSString *)path error:(NSError **)error;
```

Comment:
I think this is a MasOS 10.5 method and for this reason should be moved up to line 316.